managed-services/transactional-email/how-to/add-dmarc-record.mdx

```diff
@@ -0,0 +1,46 @@
+---
+meta:
+  title: How to configure a DMARC record
+  description: This page explains how to configure a DMARC record
+content:
+  h1: How to configure a DMARC record
+  paragraph: This page explains how to configure a DMARC record
+tags: spf spf-record dkim dkim-records transactional
+dates:
+  validation: 2024-02-21
+  posted: 2024-02-21
+categories:
+  - managed-services
+---
+
+This page shows you how to configure a [DMARC record](/managed-services/transactional-email/concepts/#dmarc-record) on your domain from the **Scaleway console**.
+If you have registered a domain at another provider, you can still use this documentation, as the process is similar.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/).
+  - You have a domain name.
+  - You have [configured your domain with Transactional Email](/managed-services/transactional-email/how-to/configure-domain-with-transactional-email).
+  - You have configured SPF and DKIM records on your domain.
+</Message>
+
+1. Click **Transactional Email** in the **Managed Services** section of the [console](https://console.scaleway.com/) side menu.
+2. Click your domain. You are redirected to the records configuration page.
+3. Add a `TXT` record type in the root zone of your domain on your DNS zone.
```

Comment:
```suggestion
3. Add a `TXT` record type in the same zone of your TEM domain on your DNS zone.
```

The domain registered at TEM is not necessarily in the root zone. 

For example, I have "mydomain.tld". And a zone "tem", I can register a TEM domain "tem.mydomain.tld" or "stg.tem.mydomain.tld".
In the first case, I'm in the root record of the "tem" zone. In the second case, I'm not a root record on the "tem" zone.
Note that this line is also in another tem how-to